lib/config/index.spec.ts

```diff
@@ -22,7 +22,8 @@ describe('config/index', () => {
     });
     it('supports token in env', async () => {
       const env: NodeJS.ProcessEnv = { RENOVATE_TOKEN: 'abc' };
-      await configParser.parseConfigs(env, defaultArgv);
+      const parsedConfig = await configParser.parseConfigs(env, defaultArgv);
+      expect(parsedConfig).toMatchSnapshot();
```

Comment:
These snapshots are picking up my local settings, e.g.

![image](https://user-images.githubusercontent.com/6311784/80786002-81eca380-8b82-11ea-9427-fdc57d7f054b.png)

I've been trying (unsuccessfully) to get the test to ignore my `config.js`

Reply:
maybe fixed now

Reply:
Thanks! I'll fix the build errors.